ClassicTotalizator/ClassicTotalizator.API/Controllers/EventsController.cs

```diff
@@ -23,10 +22,11 @@ public class EventsController : ControllerBase
         private readonly IParticipantsService _participantsService;
 
         /// <summary>
```

Comment:
Its seems that this part was implemented, but i think will be problems with authorization roles